app/shared/components/design-system/all-components/theme/Theme.ts

```diff
@@ -747,6 +748,49 @@ export const theme = createTheme({
           }
         }
       }
+    },
+    MuiAccordion: {
+      styleOverrides: {
+        root: {
+          borderRadius: '24px',
+          '&.Mui-expanded': {
+            backgroundColor: accordionColorsRgb.accordion.expanded.backgroundColor,
+            color: accordionColorsRgb.accordion.expanded.color
+          }
+        }
+      }
+    },
+    MuiAccordionSummary: {
+      styleOverrides: {
+        root: {
+          width: '742px',
+          height: '60px',
+          marginTop: '16px',
+          marginLeft: '16px',
+          justifyContent: 'space-between',
+          paddingTop: '16px',
+          paddingRight: '16px',
+          paddingBottom: '16px',
+          paddingLeft: '24px',
```

Comment:
```suggestion
          padding: '16px 16px 16px 24px'
```
If I remember correctly, you can write it shorter like this